CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+## Pull Requests
+
+Except for small changes, PRs should always address an already open and accepted
+issue. Otherwise you run the risk of spending time implementing something and
+then the PR being rejected because the feature you implemented was not actually
+something we want in db_migrator.
+
+Issues with any of the following labels are generally safe to start working on,
+unless someone else has already claimed them:
+
+* [bug]: Something isn't working
+* [enhancement]: New feature or request
+* [help wanted]: Extra attention is needed
+* [problem]: Problem getting things to work, errors and the like 
```

Comment:
That's pretty much all interesting categories.
Do we need such level of detail?

Reply:
I was inspired by another projects' CONTRIBUTING.md
It's okay to remove it

Reply:
I'm fine with `CONTRIBUTING.md` as such.
The project doesn't have the size that people are likely to get lost among the many pull requests and issues they want to work on...